src/test/java/com/thebois/models/world/WorldTests.java

```diff
@@ -43,6 +46,35 @@ public void worldFind() {
         assertThat(worldObject).isEqualTo(null);
     }
 
+    @Test
+    public void getStructuresTest() {
+        // Arrange
+        final World world = new World(25);
+
+        // Act
+        world.createStructure(new Position(123, 123));
+        final IStructure structure = world.getStructures().get(0);
+
+        // Assert
+        assertThat(structure.getPosition()).isEqualTo(new Position(123, 123));
+    }
+
+    @Test
+    public void createStructureTest() {
+        // Arrange
+        final World world = new World(25);
+
+        // Act
+        final Boolean isEmpty = world.getStructures().isEmpty();
+        final Boolean isStructureBuilt = world.createStructure(new Position(0, 0));
+        final int numberOfStructures = world.getStructures().size();
+
+        // Assert
+        assertThat(isEmpty).isTrue();
```

Comment:
Not sure if this should be part of `createStructureTest` since it only tests if we can create an empty structure world. Maybe make it a seperate test.

Reply:
Wanted to make sure the world is empty and not already containing a structure because if there was and createStructure was made incorrectly and return true this test would pass even though createStructure method doesn't work. Should I write the test in another way?

Reply:
You should have multiple tests:
`worldContainsNoStructuresFromStart()`
`numberOfStructuresGoesUpWhenStructureIsSuccessfullyPlaced()`
`numberOfStructuresDoesNotGoUpWhenStructureIsUnsuccessfullyPlaced()`

Reply:
Now that we have a test for this I think we can skip it here, right?

Reply:
^Yeah it's a bit redundant here.

Reply:
Can probably remove that test actually since they are now tested in the methods above.